tests/test_pyrflx.py

```diff
@@ -674,12 +676,12 @@ def test_parsing_ethernet_2_vlan(self) -> None:
         self.assertEqual(int("8100", 16), self.frame.get("TPID"))
         self.assertEqual(int("1", 16), self.frame.get("TCI"))
```

Comment:
The expected value should be always on the right side: `assertEqual(actual, expected)`. The file seems to be inconsistent in this matter.

Reply:
Fixed by a02992f